.github/workflows/ai-pr-review.yml

```
@@ -0,0 +1,95 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  ai_review:
+    runs-on: ubuntu-latest
+    steps:
+      # 1. PR 코드 체크아웃
+      - name: Checkout PR code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)